package.json

```diff
@@ -13,7 +13,9 @@
     "./core": {
       "import": "./dist/core.esm.js",
       "require": "./dist/core.js"
-    }
+    },
+    "./dist/assets/*.css": "./dist/assets/*.css",
```

Comment:
Did not know this works. Where can I read about it?

Reply:
My assumption to date: You explicitly have to provide the exported resources.

Reply:
Node docs at https://nodejs.org/api/packages.html#subpath-patterns

Reply:
I guess my personal preference is to still explicitly expose this. But fine with me either way.